### ISSUE TITLE
Processing time dramatically increased 

### ISSUE THREAD

**Issue by vuillaut** [cta-observatory/cta-lstchain#231]
Hi.

The processing time of real data is much longer (by about 15) than the one of MC data.
It seems that all the time goes into `pointings.py`.
@morcuended can you have a look please?
I can send you the profiling file if you want.
![image](https://user-images.githubusercontent.com/4263646/69579319-2d893a80-0fca-11ea-8cf7-25de308de97b.png)

**Comment by labsaha** [comment]
Hi @vuillaut,
Thanks for reporting. Yes, we have also seen that it is taking time. @morcuended told us about this. Now I checked the code and found misplacement of a line in the code. That is definitely a reason for that, although not sure if it is fully responsible unless we check it.

**Comment by morcuended** [comment]
Hi @vuillaut, we just went through this. We were actually opening the drive report file for each event. We fixed it so that the file is opened only once and store it in a table to find later the proper coordinates that are used to interpolate. Here is a profiling plot with the changes we made @labsaha and me (just for 100 events).

Adding pointing
![profiling_withpointing](https://user-images.githubusercontent.com/32680689/69627628-4b4ab400-104b-11ea-82bf-5f8ccc4bae4c.png)

So here the pointing part now is that pink rectangle I'm pointing at.

Without pointing
![profiling_nopointing](https://user-images.githubusercontent.com/32680689/69627266-99ab8300-104a-11ea-82a8-d812e46caf51.png)

I will open a PR with the modifications
